e2e/README.md

```diff
@@ -0,0 +1,51 @@
+# Cypress
+
+## Local setup
+On linux do not trust it to be effortless, who knows on mac?
+
+1. go to [Chromium Downloads Tool](https://chromium.cypress.io/) and download the appropriate version
```

Comment:
Do we need this step?

Reply:
Your mileage may *greatly* vary, without it, noticeable issues listed below, especially for Linux. 

It uses a variant of the v8 be it cypress via electron, or your system browser, or a custom browser.

How cypress isolated it is questionable.